department-of-veterans-affairs/caseflow
# hide SelectCheckoutFlowDropdown on cases not assigned to current user

Resolves #5519

### Description
When a user searches for an appeal, they're taken to `QueueDetailView` for that appeal, where, as long as they have `queue_phase_two` FeatureToggle enabled, they could start a checkout flow. We should prevent users from checking out cases not assigned to them.

### Acceptance Criteria 
- [ ] On Appeal detail view, Select Checkout Flow dropdown is hidden for users other than the assigned attorney

### Testing Plan
1. Go to `/queue` (as `BVASCASPER1`), search for `219228160`
2. Confirm in `QueueDetailView` that it says "Assigned to: AABSHIRE", and that the select checkout flow dropdown is hidden
